README.md

```diff
@@ -43,10 +43,12 @@ jobs:
   serial: true
   plan:
   - get: resource-web-app
+  - task: build
+    file: resource-web-app/build.yml
   - put: resource-deploy-web-app
     params:
-      manifest: resource-web-app/manifest.yml
-      path: resource-web-app
```

Comment:
Sorry for the delay in this, bit off more GitHub issues than I could chew. :)

The manifest's `path` field is always relative to the manifest itself, so it's pretty easy to prevent tight coupling with Concourse specific paths. In our case we just have a static `manifest.yml` that says `path: .`: https://github.com/concourse/concourse/blob/master/docs/manifest.yml - and then we copy that manifest into the built source.

Our build task then just produces a single output, which has the `manifest.yml` and everything else ready to go:

https://github.com/concourse/concourse/blob/master/ci/build-docs.yml
https://github.com/concourse/concourse/blob/master/ci/scripts/build-docs

So I'm not really seeing any tight coupling; it's entirely up to how you want your manifest and build script to move things around, so it should be pretty easy to have it reflect local use of the manifest.